guides/common/modules/ref_permissions-required-for-provisioning.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: REFERENCE
+
+[id="permissions-required-for-provisioning_{context}"]
+= Permissions Required for Provisioning
+
+The following table lists the permissions a non-admin user requires to provision hosts.
+
+.Permissions required for provisioning hosts
+[options="header"]
+|====
+|Resource type|Permissions
+|Compute resource|view_compute_resources, view_compute_resources_vms, create_compute_resources_vms, destroy_compute_resources_vms, power_compute_resources_vms
```

Comment:
This depends on the method that is used. For example bare-metal provisioning doesn't require `create_compute_resources_vms`

Reply:
This is about provisioning VMs, so I believe *_vms permissions should be included.